uk.co.compucorp.civicrm.hrcore/gulpfile.js

```diff
@@ -143,7 +147,15 @@ var Promise = require('es6-promise').Promise;
         return _.assign(scenario, { delay: scenario.delay || 6000 });
       })
       .tap(function (scenarios) {
-        scenarios[0].onBeforeScript = "login";
+        var previousCredential = null;
+
+        scenarios.forEach(function (scenario) {
```

Comment:
also, i think there should be some logic here that sets the default (admin) credential here if there is none defined